app/build.gradle

```diff
@@ -44,6 +44,7 @@ dependencies {
 
     implementation platform('com.google.firebase:firebase-bom:30.2.0')
     implementation 'com.google.firebase:firebase-analytics'
+    implementation 'com.google.firebase:firebase-auth-ktx:21.0.6'
```

Comment:
I think I wasn't clear enough on this. using the firebase-bom dependency from line 45 implies for you to not set the version for other firebase dependencies.

The new dependency should look like this:
```
 implementation 'com.google.firebase:firebase-auth-ktx'
```
That number `21.0.6` is the dependency version. Bom already resolves it for you

Reply:
aaa....ok . I tried to remove it first time but it doesn't work . i understand now ..thank you